### PR TITLE
S02 navbar

### PR DESCRIPTION
**Pull request by JulienOza**
navbar created
radixUI library installed
modal added for later station list display

### CODE REVIEW

**Review by SebSamPro — APPROVED** [review]

**Review by ItsMatDev — APPROVED** [review]

**Review by Bravax — APPROVED** [review]
GG

File: frontend/src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  return (
+    <nav className="navbar_container">
+      <ul className="navbar_list">
+        <li className="navbar_element">
+          <Link to="/map" draggable="false">
+            <img
+              className="navbar_img"
+              src={NavbarMap}
+              draggable="false"
+              alt="map logo"
```
Comment (Yoann-Fortin) [inline frontend/src/components/Navbar/Navbar.jsx]: Your website is in English ?

File: frontend/src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          </Link>
+        </li>
+        <li className="navbar_element navbar_element_middle">
+          <Dialog.Root>
+            <Dialog.Trigger>
+              <img
+                className="navbar_img"
+                src={NavbarStations}
+                draggable="false"
+                alt="map logo"
```
Comment (Yoann-Fortin) [inline frontend/src/components/Navbar/Navbar.jsx]: here again

File: frontend/src/components/Navbar/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            </Dialog.Portal>
+          </Dialog.Root>
+        </li>
+        <li className="navbar_element">
+          <Link to="/" draggable="false">
+            <img
+              className="navbar_img"
+              src={NavbarUserPage}
+              draggable="false"
+              alt="map logo"
```
Comment (Yoann-Fortin) [inline frontend/src/components/Navbar/Navbar.jsx]: and here

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: frontend/src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (JulienOza) [reply frontend/src/components/Navbar/Navbar.jsx]: With all that coding in English, dailies in English, .... I guess I turned a little bit "Jean-Claude Vandamme"
It's fixed in the last commit !

File: frontend/src/components/Navbar/Navbar.jsx (hunk 2 above)
Comment (JulienOza) [reply frontend/src/components/Navbar/Navbar.jsx]: fixed in the last commit

File: frontend/src/components/Navbar/Navbar.jsx (hunk 3 above)
Comment (JulienOza) [reply frontend/src/components/Navbar/Navbar.jsx]: fixed in the last commit

**Pull request closed without merging**